packages/typespec-go/test/armlargeinstance/zz_azurelargeinstance_client.go

```diff
@@ -16,23 +16,23 @@ import (
 	"strings"
 )
 
-// AzureLargeInstancesClient contains the methods for the AzureLargeInstances group.
-// Don't use this type directly, use NewAzureLargeInstancesClient() instead.
-type AzureLargeInstancesClient struct {
+// AzureLargeInstanceClient contains the methods for the AzureLargeInstance group.
```

Comment:
Is this change expected? In https://github.com/Azure/autorest.go/blob/main/packages/typespec-go/test/tsp/AzureLargeInstance.Management/AzureLargeInstance.tsp#L28 I see the interface name is `AzureLargeInstances`.

Reply:
it is expected. tcgc fixed a bug of not seeing the `@clientName` on interface which made the previous generated code with `s`: https://github.com/Azure/autorest.go/blob/ea240ed75fc1603fd21fed6017eede252eed6d76/packages/typespec-go/test/tsp/AzureLargeInstance.Management/AzureLargeInstance.tsp#L27